### ISSUE TITLE
[Bug] <Write the microsd card to the storagelimit>

### ISSUE THREAD

**Issue by WzyErik** [tachiyomiorg/tachiyomi#4323]
At first, I'm sorry for my low school English skill. Germany has nice English teachers. 

## Device information
* Tachiyomi version: 0.10.7
* Android version: 10 1.Jan.2021
* Device: Samsung Galaxy S9 Duos

## Steps to reproduce

In first situation, I change my location  (from home to work and I were downloading some chapters at home).
My smartphone reconnected itself to the workplace wifi. 
After sometimes I got an alert, that my SanDisk Ultra 200 GB microsd card reached the storage limit and need to be replaced (RIP).
So I replaced with a new Samsung evo plus 256 GB microsd card.

In the second situation, I were downloading many of chapters. 

### Expected behavior

Downloaded chapters without any errors. 

### Actual behavior

Write one of my microsd to the dead and other one to the storage limit.

## Other details

I don't know, because I'm just downloading chapters (I think it was about 100, however it was many.).

**Comment by arkon** [comment]
Are you saying you didn't actually download enough to fill up the SD card?

**Comment by WzyErik** [comment]
> Are you saying you didn't actually download enough to fill up the SD card?

Yup, I checked my microsd card with my pc. It said, 10 GB was described, but 239 GB (the real storage limit of a 256 GB card) was occupied.

**Comment by arkon** [comment]
Sorry, I'm having difficulties understanding you. Could you also write your issue in German? I can have someone translate that.

**Comment by WzyErik** [comment]
Okay, das Problem ist, dass Tachiyomi mein MicroSD voll schreibt. Ich war am Chapter downloaden, bemerkte dann, dass irgendein der Chapter korrupt war. Sprich in Wirklichkeit war es nur z.B. 10 MB groß, aber würde so groß dargestellt, dass es mein MicroSD überfüllt hat.

**Comment by arkon** [comment]
> Okay, the problem is that Tachiyomi is filling up my MicroSD. I was downloading chapters, then noticed that any of the chapters were corrupt. Say, in reality, it was only, say, 10 MB, but would display so large that it overfilled my MicroSD.

(Mix of DeepL and human translation)

**Comment by arkon** [comment]
Probably related: #3373

**Comment by WzyErik** [comment]
I didn't have the "folder name to long" error, but maybe.
